### DISCUSSION TITLE
Bromite for linux

### DISCUSSION THREAD

**Discussion by Alex313031** [bromite/bromite#1404] [General]
Hello, I am creating a fork for linux, and I would like to include the bromite logo, but wish to edit it to be black, blacker, blackest, rather than green, greener, greenest. Anyway, I'm asking permission even though its GPL3, but also the dirs related to logo change seem to be missing from your repo. Where can I get this, or if needed I'll do it myself, but I would like the highest res, or uncompressed/lossless, or 8x8...16x16...32x32...etc. icon set and any related UI changes/icon changes within bromite.

As needed changes would be made to build properly on linux and I will be including widevine and all thinlto and pgo optimizations.
@spiderman973 @csagan5

**Comment by csagan5** [comment]
This is the third post you have made about this.

The logo is not covered by GPLv3 and you do not have permission to call your fork Bromite, since it will not be official.

**Reply by Alex313031** [reply]
Im sorry, the other posts said missing template so I assumed it didn't get through. I made the comment on bromite for desktop because it seemed fitting. I wasn't going to call it bromite, and okay thanks for specifying on the logo. I might make my own or just leave the pretty chromium logo.

**Reply by csagan5** [reply]
You might as well use https://github.com/Eloston/ungoogled-chromium as a base, since it contains already some Bromite patches.

**Comment by Oymate** [comment]
@Alex313031 you can call it any other name and logo, that would probably a better way to do it.
